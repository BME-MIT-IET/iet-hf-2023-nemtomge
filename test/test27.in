loadmap map.json  
start V1  
set Scientist V1 PL1  
give Crafted V1 Stun 1 
give Crafted V1 Craziness 1 
use V1 V1 
1
has Active V1 Stun 
use V1 V1 
show Active V1
exit
