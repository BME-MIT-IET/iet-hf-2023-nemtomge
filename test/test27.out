Map from file map.json loaded
The game has been started. Players are: V1   
Scientist V1 set on field PL1 
Scientist V1 has been given crafted agent Stun time:  1 
Scientist V1 has been given crafted agent Craziness time:  1 
Choose one to use: 
0 - cancel 
1 - Stun(1) 
2 - Craziness(1) 
Scientist V1 has active agent Stun time: 1
V1's active effects are:
Stun(1)
